packages/app/src/app/pages/common/LikeHeart/index.tsx

```diff
@@ -17,11 +17,11 @@ type Props = {
   colorless?: boolean;
   sandbox: Sandbox;
   text?: number;
-} & Partial<Pick<ComponentProps<typeof MaybeTooltip>, 'disableTooltip'>> &
-  Pick<
-    ComponentProps<typeof Container>,
-    'className' | 'highlightHover' | 'style'
-  >;
+  className?: string;
```

Comment:
@SaraVieira Why did you change this back? 🤔
I used `Pick` so it's more clear what props we just pass through to the `Container` & `MaybeTooltip` components